contributor-docs/migrating-to-css-modules.md

```diff
@@ -121,11 +120,65 @@ This guide outlines the steps to follow when refactoring Primer React components
       expect(render(<FeatureFlagElement />).container.firstChild).toHaveClass('test-class-name')
     })
     ```
-- **Regression Testing:**
-  - Validate that no visual regressions occur when the feature flag is enabled. The `vrt*` tests are setup to compare the feature flagged component with the original component and will fail if there is a mismatch.
 - **Handling `sx` Prop:**
   - Confirm the `sx` prop behaves correctly with the feature flag enabled.
 
+#### Visual regression testing
+
+Validate that no visual regressions occur when the feature flag is enabled. The `vrt*` tests are setup to compare the feature flagged component with the original component and will fail if there is a mismatch.
```

Comment:
Seems like a good idea! Would we want to make sure we mark them to be removed once the CSS module migration is complete?

Reply:
I think we want to keep the dev tests around because they are testing for `className` override support mainly. When the feature flags are all removed we will then remove the check for enabled vs. default, but we can do that all at once probably since each test uses the same configuration